Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -251,5 +272,26 @@ void OnDestroy()
             InteractionManager.SourceUpdated -= InteractionManager_SourceUpdated;
             InteractionManager.SourceLost -= InteractionManager_SourceLost;
         }
+
+        public void OnSelect()
+        {
+            ThrowDeprecatedWarning();
```

Comment:
Are you referring to the scenario where your existing code will fire off deprecation warnings? If so, fix your code.

That said, this doesn't look like deprecation. Looks like obsolescence, would recommend renaming this appropriately and/or just nixing the API and letting the compile errors surface. (With appropriate documentation of course)